makefile

```diff
@@ -16,7 +16,8 @@ WORKLOAD =SSBM
 # add -EDTERABYTE if orderkey will execeed 32 bits (SF >= 300)
 # and make the appropriate change in gen_schema() of runit.sh
 # HYRISE: Change -O to -O3. Add -Wno-dangling-else, -Wno-int-to-void-pointer-cast, and -Wno-void-pointer-to-int-cast because they are all over the codebase.
-CFLAGS	= -O3 -DDBNAME=\"dss\" -D$(MACHINE) -D$(DATABASE) -D$(WORKLOAD) -Wno-dangling-else -Wno-int-to-void-pointer-cast -Wno-void-pointer-to-int-cast -Wno-pointer-to-int-cast -Wno-int-to-pointer-cast
+CFLAGS ?=
```

Comment:
What does that do?

Reply:
Sets the `CFLAGS` to empty string or copies them if they are set by an environment variable. In the next line, our regular flags are added. Thus, we can add `-Werror` in the CI but omit it in the Hyrise built.